alert/log/entry.go

```diff
@@ -152,6 +154,22 @@ func (e Entry) String() string {
 	return msg
 }
 
+func (e Entry) Details() string {
+	return e.statusDetails.String
+}
+
+func (e Entry) Status() string {
+	if e.lastStatus.String == "failed" {
+		return "ERROR"
+	}
+
+	if e.lastStatus.String == "delivered" {
+		return "OK"
+	}
+
+	return ""
+}
```

Comment:
Since these are specific for only one type of Entry, I wonder if there's a better way we can do this. What would the `Status` of an escalation be for instance?

We could just have one method `OutgoingMessageState` that returns a struct or nil with `StatusDetails` and `LastStatus` (since it only has value when there is a message id). I'd also wait until GraphQL to map to `ERROR` or `OK` and keep this part plain as possible.

Alternatively, we could also leave Entry untouched, and do a separate query for the state (e.g. with a datafetcher using the ID from meta). That would have the benefit that we wouldn't always be joining to the messages table. We would only access it if there was a log entry with a message ID in the results.

The former would probably be the most straightforward and resolves the ambiguity. Though the latter would probably cleanest from an API standpoint (not adding arbitrary fields/methods and keeping everything general on Entry).

Reply:
> Alternatively, we could also leave Entry untouched, and do a separate query

Would there be any limitations to this? After the first page we load the next 150 logs, and if many of those have their state set, that's a lot of extra trips to the DB.

Reply:
The datafetcher would batch them together, so it would only be one query for all 150 worst case (though not all 150 would be `notification_sent`).

Reply:
Let's try leaving Entry untouched and grab the data as a separate query, then